roles/engine_setup/README.md

```diff
@@ -41,6 +41,7 @@ to ``ovirt_engine_setup_answer_file_path`` variable.
 | ovirt_engine_setup_wait_running_tasks | False                 | If `True`, engine-setup will wait for running tasks to finish. Valid for `ovirt_engine_setup_version` >= 4.2. |
 | ovirt_engine_cinderlib_enable         | False                 | If `True`, cinderlib is enabled. Valid for `ovirt_engine_setup_version` >= 4.3. |
 | ovirt_engine_grafana_enable           | True                  | If `True`, Grafana integration will be set up. Valid for `ovirt_engine_setup_version` >= 4.4. |
+| ovirt_engine_setup_skip_renew_pki_confirm | 'yes'                | If `yes` PKI renewal will be skipped
```

Comment:
There are sometimes issues with ansible using `'yes'` because ansible interprets this as `True`, similarly `'no'` is interpreted as `False`. For this reason, I showed the example of `ovirt_engine_apache_config_root_redirection`